_books/ion-1-1/src/modules/shared_modules.md

```diff
@@ -4,7 +4,7 @@ Shared modules exist independently of the documents that use them.
 They are identified by a _catalog key_ consisting of a string name and an integer version.
 
 The self-declared catalog-names of shared modules are generally long, since they must be more-or-less globally unique.
-When imported by another module, they are given local symbolic names by import declarations.
+When imported by another module, they are given local symbolic names (a "binding") by import declarations.
```

Comment:
Italicize to indicate that it's a technical term that is (or will be) in the glossary.
```suggestion
When imported by another module, they are given local symbolic names (a _binding_) by import declarations.
```